deploy/kubernetes/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pullsheet
+spec:
+  selector:
+    matchLabels:
+      app: pullsheet
+  template:
+    metadata:
+      labels:
+        app: pullsheet
+    spec:
+      containers:
+      - name: pullsheet
+        image: pullsheet
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+        env:
+        - name: PULLSHEET_REPOS
+          valueFrom:
```

Comment:
Can you remove a level of indirection here? I don't think the ConfigMap adds much value here. Instead of `valueFrom`, just say: `value: "google/pullsheet"`

Reply:
For reference: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

Reply:
good point! I guess my mind just wanted some kind of separate file for the config, but it doesn't really make sense right now. I'd like to make a follow up after this that uses the viper library to actually read from a config file as well.